sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeFixedWidthAggregationMap.java

```diff
@@ -55,9 +67,19 @@
   private final BytesToBytesMap map;
 
   /**
+   * A object pool for these in keys.
+   */
+  private final UniqueObjectPool keyPool;
+
+  /**
+   * A object pool for those in aggregation buffers.
+   */
+  private final ObjectPool bufferPool;
```

Comment:
The type is already `ObjectPool`,  having the full name in the variable name is too verbose, I think.